atomix/cluster/src/main/java/io/atomix/raft/protocol/PersistedRaftRecord.java

```diff
@@ -60,6 +60,11 @@ public DirectBuffer data() {
     return new UnsafeBuffer(serializedRaftLogEntry);
   }
 
+  @Override
+  public DirectBuffer serializedRecord() {
+    return null;
```

Comment:
:wrench: imo this may lead to more confusion than anything, so I would `throw UnsupportedOperationException()` here instead, or implement it.